libs/algebra/include/nil/crypto3/algebra/curves/detail/forms/edwards/inverted/madd_2007_bl.hpp

```diff
@@ -44,6 +43,10 @@ namespace nil {
                         constexpr static inline void process(ElementType &first,
                                                                     const ElementType &second) {
 
+                            /*  NOTE: does not handle O and pts of order 2,4, i.e.
```

Comment:
Maybe move these back above the class definition, where it was. But keep the new text, it's more clear now.